### ISSUE TITLE
WKID question of OGCGeometry.fromText() and OGCGeometry.fromBinary()

### ISSUE THREAD

**Issue by yiranwang** [Esri/geometry-api-java#139]
For the function: 
public static OGCGeometry fromText(String text) {
        OperatorImportFromWkt op = (OperatorImportFromWkt)OperatorFactoryLocal.getInstance().getOperator(Type.ImportFromWkt);
        OGCStructure ogcStructure = op.executeOGC(0, text, (ProgressTracker)null);
        return createFromOGCStructure(ogcStructure, SpatialReference.create(4326));
    }
For this function, it will output OGCGeometry, it use the WKID 4326 directly. If the input text has some information of SRID which is not 4326, what should we do? or current api for all the relations (such as st_contains, st_intersects etc.) is just suited for 4326 spatial reference system.

**Comment by alocke** [comment]
Currently, ImportFromWkt doesn't import spatial references from text. We can add that functionality if you need it. 
I don't understand what you mean when you say that relational operations are suited only for 4326. Can you explain it more?

**Comment by yiranwang** [comment]
Thank you so much! Yes, I think for the functions fromText or fromBinary, it's better to have the part to import the spatial reference. If there is no information of WKID or SRID in the WKT or WKB, we could use the 4326 as default one. For the second question, for example, I run this:
**containsOperator.execute(leftGeometry.getEsriGeometry(), rightGeometry.getEsriGeometry(), leftGeometry.getEsriSpatialReference(), null);** 
    In public boolean execute(Geometry inputGeom1, Geometry inputGeom2, SpatialReference sr, ProgressTracker progressTracker) {
        return RelationalOperations.relate(inputGeom1, inputGeom2, sr, 1, progressTracker);
    } 
In sr, if the WKID is not 4326, could we guarantee the correctness of thecontainsOperator.execute(...)?

**Comment by alocke** [comment]
Yes, the relational operators work correctly no matter which spatial reference is used.
